app/views/layouts/_nav.html.erb

```diff
@@ -1,11 +1,29 @@
-<nav class="navbar navbar-light bg-light">
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
   <a class="navbar-brand" href="/">
     <%= image_tag("logo.png", alt: "BookLand", width: "100") %>
     <h1 class="d-inline">BookLand!</h1>
   </a>
-  <% if user_signed_in? %>
-    <%= link_to("Logout", destroy_user_session_path, method: :delete, class: "btn btn-success") %>
-  <% else %>
-    <%= link_to("Login", new_user_session_path, class: "btn btn-success")  %>
-  <% end %>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse justify-content-end" id="navbar">
+    <% if user_signed_in? %>
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <%= link_to("Users", users_path, class: "nav-link") %>
+        </li>
+        <li class="nav-item">
+          <%= link_to("Books", books_path, class: "nav-link") %>
+        </li>
+        <li class="nav-item">
```

Comment:
Upon reviewing the BS4 docs, I remembered that these nav items need to be in a `<ul>`.

Does this make sense to put the `<li>` in this layout, and then the individual link in the new nested layouts?